Documentation/Tools/ControllerMappingTool.md

```diff
@@ -0,0 +1,37 @@
+# Controller Mapping Tool
+
+The controller mapping tool is a runtime (on device or in the editor) tool that enables developers to quickly determine the Unity input axis and button mappings for a hardware controller (ex: motion controller).
+
+THis tool is very useful when developing support for a new hardware controller. It can also help to confirm a suspected control mapping issue in the support class for an existing controller.
```

Comment:
```suggestion
This tool is very useful when developing support for a new hardware controller. It can also help to confirm a suspected control mapping issue in the support class for an existing controller.
```